core/server/models/member.js

```diff
@@ -106,6 +106,11 @@ const Member = ghostBookshelf.Model.extend({
     onSaving: function onSaving(model, attr, options) {
         let labelsToSave = [];
 
+        if (_.isUndefined(this.get('labels'))) {
+            this.unset('labels');
```

Comment:
This is to ensure that an undefined value is treated the same as the absence of a value e.g.
```js
Member.add({name: 'egg', email: 'egg@monst.er'})
Member.add({name: 'egg', email: 'egg@monst.er', labels: undefined})
```
From what I can tell, both calls will be skipped over by `bookshelf-relations` and then inserted - however the second call needs to be looked at by bookshelf relations, so that it can have the property removed before being sent to the database! 

This change ensures that consumers of the Model have a more understandable interface

Reply:
Makes sense 👍